roles/ceph-grafana/templates/datasources-ceph-dashboard.yml.j2

```diff
@@ -17,7 +17,7 @@ datasources:
   # <int> org id. will default to orgId 1 if not specified
   orgId: 1
   # <string> url
-  url: 'http://{{ groups["grafana-server"][0] }}:9090'
+  url: 'http://{{ ansible_default_ipv4.address }}:9090'
```

Comment:
Could we use something more flexible ?
Because using the default ipv4 address will probably work in most cases but not always (different networks, ipv6, etc..)